packages/kbn-plugin-generator/integration_tests/generate_plugin.test.js

```diff
@@ -78,16 +79,7 @@ describe(`running the plugin-generator via 'node scripts/generate_plugin.js plug
       });
     });
 
-    it(`'yarn test:server' should exit 0`, async () => {
-      await execa('yarn', ['test:server'], {
-        cwd: generatedPath,
-        env: {
-          DISABLE_JUNIT_REPORTER: '1',
-        },
-      });
-    });
-
-    it(`'yarn build' should exit 0`, async () => {
+    it.skip(`'yarn build' should exit 0`, async () => {
       await execa('yarn', ['build'], { cwd: generatedPath });
     });
```

Comment:
I see the comment about the i18n test, what about the `yarn build` and `yarn lint`. If they are obsolete they should probably be removed instead of skipped?